src/components/SimpleCard/SimpleCard.stories.ts

```diff
@@ -1,9 +1,22 @@
 import type { Meta, StoryObj } from "@storybook/web-components-vite"
 import { html } from "lit"
 import { updateShopifyRoot } from "../../utils/storybook"
+import { getExampleHandles } from "../../shopify/graphql/getExampleHandles"
 
 const root = "https://nosto-shopify1.myshopify.com/"
-const handles = ["good-ol-shoes", "awesome-sneakers", "old-school-kicks", "insane-shoes"]
+// Fallback handles in case the API call fails
+const fallbackHandles = ["good-ol-shoes", "awesome-sneakers", "old-school-kicks", "insane-shoes"]
+
+let handles: string[]
+try {
+  handles = await getExampleHandles(root, 12)
```

Comment:
does storybook support async stories in a way that we could move this invocation to happen inside the story object?